src/Primitives/src/StringSegment.cs

```diff
@@ -326,9 +326,9 @@ public bool StartsWith(string text, StringComparison comparisonType)
         /// </summary>
         /// <param name="text">The <see cref="string"/>to compare.</param>
         /// <param name="comparisonType">One of the enumeration values that specifies the rules to use in the comparison.</param>
-        /// <returns><code>true</code> if <paramref name="text"/> matches the end of this <see cref="StringSegment"/>; otherwise, <code>false</code>.</returns>
+        /// <returns><c>true</c> if <paramref name="text"/> matches the end of this <see cref="StringSegment"/>; otherwise, <c>false</c>.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true" /> if <paramref name="text"/> matches the end of this <see cref="StringSegment"/>; otherwise, <see langword="false" />.</returns>
```